opentelemetry-exporter-geneva/geneva-uploader/src/payload_encoder/otlp_encoder.rs

```diff
@@ -251,35 +237,14 @@ impl OtlpEncoder {
         (field_defs, schema_id)
     }
 
-    /// Get or create schema - fields are accessible via returned schema entry
+    /// Create schema - always creates a new CentralSchemaEntry
     fn get_or_create_schema(&self, schema_id: u64, field_info: &[FieldDef]) -> CentralSchemaEntry {
```

Comment:
Do you think it's better to remove the `&self` parameter from these methods: `encode_log_batch`, `write_row_data`?

```suggestion
    fn create_schema(schema_id: u64, field_info: &[FieldDef]) -> CentralSchemaEntry {
```